demisto_sdk/commands/common/hook_validations/script.py

```diff
@@ -220,3 +223,27 @@ def is_valid_as_deprecated(self) -> bool:
                 if self.handle_error(error_message, error_code, file_path=self.file_path):
                     is_valid = False
         return is_valid
+
+    def is_valid_script_file_path(self) -> bool:
+        absolute_file_path = self.file_path
+        scripts_folder = os.path.basename(os.path.dirname(absolute_file_path))
+        script_file = os.path.basename(absolute_file_path)
+        script_file, _ = os.path.splitext(script_file)
+
+        if scripts_folder == 'Scripts':
+            if not script_file.startswith('script-'):
+
+                error_message, error_code = Errors.is_valid_script_file_path_in_scripts_folder(script_file)
+                if self.handle_error(error_message, error_code, file_path=self.file_path):
+                    return False
+            return True
```

Comment:
drop this return and change the if in 241 to elif

Reply:
done